[2022-12-04T16:14:32.584+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-04T10:00:00+00:00 [queued]>
[2022-12-04T16:14:32.591+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-04T10:00:00+00:00 [queued]>
[2022-12-04T16:14:32.592+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:32.594+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:14:32.595+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:32.604+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): generate_data> on 2022-12-04 10:00:00+00:00
[2022-12-04T16:14:32.610+0000] {standard_task_runner.py:55} INFO - Started process 567 to run task
[2022-12-04T16:14:32.617+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'generate_data', 'scheduled__2022-12-04T10:00:00+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmp2r08idlk']
[2022-12-04T16:14:32.624+0000] {standard_task_runner.py:83} INFO - Job 71: Subtask generate_data
[2022-12-04T16:14:32.692+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.generate_data scheduled__2022-12-04T10:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:14:32.758+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=generate_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T10:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-04T10:00:00+00:00
[2022-12-04T16:14:32.953+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-04T16:14:32.980+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=generate_data, execution_date=20221204T100000, start_date=20221204T161432, end_date=20221204T161432
[2022-12-04T16:14:33.031+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:14:33.056+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
